meetings-meetups/_posts/2020-01-16-foss-hours.md

```diff
@@ -0,0 +1,10 @@
+---
+layout: redirect
+redirect: https://fossrit.github.io/
+date-start: "2020-01-16 17:00"
+date-end: "2020-01-16 19:00"
+location: "MSS-3190"
+title: "FOSS Hours"
+rrule: "FREQ=WEEKLY;UNTIL=20200423T190000"
```

Comment:
What is the syntax for this frontmatter? I want to link out to documentation about how to write these for future contributors.

Reply:
@jwflory https://github.com/RITlug/ritlug.github.io#calendar--placeholders

> Placeholders can also take an `rrule` frontmatter for recurring events, written [to the iCal spec](https://icalendar.org/iCalendar-RFC-5545/3-3-10-recurrence-rule.html).
>
> Of particular note,
> 
> > The UNTIL rule part defines a DATE or DATE-TIME value that bounds the recurrence rule in an inclusive manner. If the value specified by UNTIL is synchronized with the specified recurrence, this DATE or DATE-TIME becomes the last instance of the recurrence. The value of the UNTIL rule part MUST have the same value type as the "DTSTART" property.

Reply:
@ct-martin Thanks. I'll use that as a base for the Runbook.